detectors/resources/src/main/java/com/google/cloud/opentelemetry/detectors/GCPResource.java

```diff
@@ -184,8 +184,9 @@ private void assignGKEAvailabilityZoneOrRegion(
         attributesBuilder.put(ResourceAttributes.CLOUD_AVAILABILITY_ZONE, clusterLocation);
         break;
       default:
-        attributesBuilder.put(ResourceAttributes.CLOUD_REGION, clusterLocation);
-        attributesBuilder.put(ResourceAttributes.CLOUD_AVAILABILITY_ZONE, clusterLocation);
+        // TODO: Figure out how to handle unexpected conditions like this - Issue #183
+        attributesBuilder.put(ResourceAttributes.CLOUD_REGION, "");
```

Comment:
I wouldn't bother putting empty attributes in.  Seems like it would just be confusing.